File: backend/team6/src/main/java/programmers/team6/global/config/WebConfig.java
package programmers.team6.global.config;

import java.util.List;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.method.support.HandlerMethodArgumentResolver;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

import lombok.RequiredArgsConstructor;
import programmers.team6.domain.member.util.LoginMemberArgumentResolver;
import programmers.team6.global.paging.CustomPageableHandlerMethodArgumentResolver;

@Configuration
@RequiredArgsConstructor
public class WebConfig implements WebMvcConfigurer {

	private final LoginMemberArgumentResolver loginMemberArgumentResolver;

	@Override
	public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
		resolvers.add(new CustomPageableHandlerMethodArgumentResolver());
		resolvers.add(loginMemberArgumentResolver);
	}

}

File: backend/team6/src/main/java/programmers/team6/global/config/WebConfig.kt
package programmers.team6.global.config

import lombok.RequiredArgsConstructor
import org.springframework.context.annotation.Configuration
import org.springframework.web.method.support.HandlerMethodArgumentResolver
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
import programmers.team6.domain.member.util.LoginMemberArgumentResolver
import programmers.team6.global.paging.CustomPageableHandlerMethodArgumentResolver

@Configuration
class WebConfig(
    private val loginMemberArgumentResolver: LoginMemberArgumentResolver
) : WebMvcConfigurer{
    override fun addArgumentResolvers(resolvers: MutableList<HandlerMethodArgumentResolver>) {
        resolvers.add(CustomPageableHandlerMethodArgumentResolver())
        resolvers.add(loginMemberArgumentResolver)
    }
}
